lib/welcome_screen.dart

```diff
@@ -5,6 +5,7 @@ import 'package:camera/camera.dart';
 import "camera_screen.dart";
 import 'screen_button.dart';
 import "menu_screen.dart";
+import "instruction_screen.dart";
```

Comment:
You seem to have deleted the code that navigated to the instructions screen when merging updates from main.